src/library.js

```diff
@@ -3684,6 +3684,7 @@ DEFAULT_LIBRARY_FUNCS_TO_INCLUDE.push(
   '$addFunction',
   '$removeFunction',
   '$allocate',
+  '$ALLOC_STACK',
```

Comment:
Why just `ALLOC_STACK` and not the other `ALLOC_`?

Reply:
Good point, I added the other. I just included that one because it was used in test.